src/gui.c

```diff
@@ -1691,10 +1691,16 @@ send_hover_event (const gchar *uri, const gchar *title)
 void
 set_window_property (const gchar *prop, const gchar *value)
 {
-    if (uzbl.gui.main_window && GTK_IS_WIDGET (uzbl.gui.main_window))
-    {
+    GdkWindow *window;
+
+    if (uzbl.gui.main_window && GTK_IS_WIDGET (uzbl.gui.main_window)) {
+        if (!(window = gtk_widget_get_window (uzbl.gui.main_window))) {
```

Comment:
I fixed this on `next`. Couldn't remember where I had seen it before and why the patch wasn't applied :/ . See cec53db6685b15ed2130f04792013f22998af10e.